.gitignore

```diff
@@ -5,5 +5,4 @@ config.js
 .firebaserc
 firebase.json
 .firebase/
-logs
-.vscode
\ No newline at end of file
+logs
```

Comment:
Newline needed.
Same for all files.
Update your editor to include newline at end of file everytime you save.